tools/pyclient/src/molgenis_emx2_pyclient/client.py

```diff
@@ -14,27 +21,29 @@ def __init__(self, url: str, username: str = None, password: str = None) -> None
         A Client class instances is created with a server url, username and password.
         The object starts a network session and logs in to the server using the login credentials.
         """
-        self.url = self.__parse_url(url)
+        self.url = utils.parse_url(url)
+        self.api_graphql = self.url + "/apps/central/graphql"
         self.signin_status = 'unknown'
 
+        self.schemas: list = []
+
         self.session = requests.Session()
 
+        # Sign in when user credentials are supplied
         if username is not None and password is not None:
-            self.signin(username, password)
+            self.sign_in(username, password)
```

Comment:
if this is built closely on graphql then I would recommend we really mirror the function names. 
(p.s. can we actually generate a reasonable python client from graphql schema or doesn't that help at all?)

Reply:
Sign in and sign out have been renamed